features/ultils/helper.ts

```diff
@@ -48,6 +54,22 @@ const replacer = (substring: string) => {
   return value;
 };
 
+const worldEnvReplacer = (
+  substring: string,
+  envVars: { [key: string]: string } | undefined,
+) => {
+  if (!envVars) {
+    return substring;
+  }
+
+  const key = substring.replace("$", "");
+  const value = envVars[key];
+  if (value === undefined) {
+    throw new Error(`The env variable is missing: ${key}`);
```

Comment:
NITS:
The error message here is better to be different from that of processEnvReplacer.

```suggestion
    throw new Error(`The env variable in World context is missing: ${key}`);
```

Reply:
NITS:
BTW, is it `worldEnv`? 
Because it is in `utils`, do we need to choose more no-context naming like `inputEnv`?